### ISSUE TITLE
Camera moves according to head position in seated mode

### ISSUE THREAD

**Issue by Heli0saurus** [Eusth/IOVR#1]
Double clicking CTRL+C makes the camera look like it just injected every form of meth and spins faster the further your head turned from your original position(? i think) when you opened the game. Pro-tip if you try it, **don't hit f5 it makes it so, so much worse.**

**Comment by Eusth** [comment]
Yes, that's a bug I actually already fixed. Will be in the next release.




Written on mobile







On Fri, Oct 12, 2018 at 3:45 PM +0200, "Heli0saurus" <notifications@github.com> wrote:












Double clicking CTRL+C makes the camera look like it just injected every form of meth and spins faster the further your head turned from your original position(? i think) when you opened the game. Pro-tip if you try it, don't hit f5 it makes it so, so much worse.
